docs/api_reference.rst

```diff
@@ -9,13 +9,40 @@ methods in the current release of PyMC experimental.
   :maxdepth: 2
 
 
+:mod:`pymc_experimental`
+=============================
+
+.. automodule:: pymc_experimental.marginal_model
+   :members: MarginalModel
```

Comment:
This is probably the most important comment. I am not sure how I am supposed to import the MarginalModel class.

It will be documented here as `pymc_experimental.marginal_model.MarginalModel`, but it is also inside the "pymc_experimental" section which makes me want to try using `pymc_experimental.MarginalModel`

Reply:
For now I would emphasize/require the verbose imports